modules/apps/dynamic-data-mapping/dynamic-data-mapping-expression/src/main/java/com/liferay/dynamic/data/mapping/expression/internal/DDMExpressionEvaluatorVisitor.java

```diff
@@ -203,8 +203,8 @@ public Object visitFunctionCallExpression(
 				_ddmExpressionFieldAccessor);
 		}
 
-		Optional<Method> methodOptional = _getExpressionFunctionApplyMethod(
-			ddmExpressionFunction);
+		Optional<Method> methodOptional =
+			_getOptionalExpressionFunctionApplyMethod(ddmExpressionFunction);
```

Comment:
Hey @marcosapmf, this is not following [the rule mentioned](https://github.com/brianchandotcom/liferay-portal/pull/87963#issuecomment-619323559) by Brian, can you please re-send? Thanks.